templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ view.object.pk|yesno:"Update Manufacturer,Create New Manufacturer" }}</h1>
+
+  <form method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <button type="submit" class="btn btn-success">Save</button>

```

Comment:
Consider adding a `Cancel` button to improve user experience. This button should redirect the user to the previous page or a relevant list page.